graphql_compiler/schema_transformation/make_query_plan.py

```diff
@@ -0,0 +1,274 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import namedtuple
+from copy import copy
+
+from graphql import print_ast
+from graphql.language import ast as ast_types
```

Comment:
heh I guess you wrote a lot of code that had this import line? :)

Reply:
Oops, I meant to fix this before pushing and then forgot